Documentation/EyeTracking/EyeTracking_TargetSelection.md

```diff
@@ -38,14 +77,16 @@ color when being looked at.
     }
 ```
 
-#### Selecting a Focused Hologram 
-To select focused holograms, use input event listeners to confirm a selection. 
+### Selecting a focused hologram 
+To select focused holograms, use PointerHandler to listener for input event to confirm a selection. 
 For example, adding the _IMixedRealityPointerHandler_ will make them react to simple pointer input. 
 The _IMixedRealityPointerHandler_ interface requires implementing the following three interface members: 
 _OnPointerUp_, _OnPointerDown_, and _OnPointerClicked_.
 
-The _MixedRealityInputAction_ is a configurable list of actions that you want to distinguish in your app and can be edited in the 
-_MRTK Configuration Profile_ -> _Input System Profile_ -> _Input Actions Profile_. 
+In the example below, we change the color of a hologram by looking at it and pinching or saying "select".
+Which action is required to trigger the event is defined by ```eventData.MixedRealityInputAction == selectAction``` whereby we can set the type of ```selectAction``` in the Unity Editor - by default it's the "Select" action.
+The types of available [MixedRealityInputActions](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Input/InputActions.html) can be configured in the MRTK Profiler via
```

Comment:
```suggestion
The types of available [MixedRealityInputActions](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Input/InputActions.html) can be configured in the MRTK Profile via
```

Reply:
The weird part is that a profiler is different from a profile in a fairly significant way i.e. perf profiler vs configuration

Reply:
Oh, is it? I don't even know. Good catch then!